Picard/2.26.4--hdfd78af_0/CollectArraysVariantCallingMetrics.nf

```diff
@@ -0,0 +1,33 @@
+process CollectArraysVariantCallingMetrics {
+    tag {"CollectArraysVariantCallingMetrics ${identifier}"}
+    label 'PICARD_2_26_4_hdfd78af_0'
+    label 'PICARD_2_26_4_hdfd78af_0_CollectArraysVariantCallingMetrics'
+    container = 'quay.io/biocontainers/picard:2.26.4--hdfd78af_0'
```

Comment:
Maybe we should use broad (developer) dockers for picard as well? 

https://hub.docker.com/r/broadinstitute/picard